templates/definition/meter/huawei-emma.yaml

```diff
@@ -0,0 +1,126 @@
+template: huawei-emma
+products:
+  - brand: Huawei
+    description:
+      generic: EMMA
+params:
+  - name: usage
+    choice: ["grid", "pv", "battery"]
+    allinone: true
+  - name: storageunit
```

Comment:
Wozu brauchts das?

Reply:
Wird dies für das Battery Control gebraucht? Dann würde ich es noch drin lassen und mich danach and die Steuerung setzen.

Reply:
Wenn es nicht benutzt wird bitte raus